tensorflow/core/util/tensor_bundle/byte_swap.h

```diff
@@ -108,6 +109,19 @@ Status ByteSwapArray(char *array, size_t bytes_per_elem, int array_len);
 // TODO(frreiss): Should this be a member of the Tensor class?
 Status ByteSwapTensor(Tensor *t);
 
+// Byte-swap a buffer in place.
+//
+// Args:
+//  buff: pointer to the buffer to be modified IN PLACE.
+//  size: size of bytes in this buffer.
+//  dtype: type of data in this buffer.
+//  num_of_elem: number of data in this buffer, set to -1 if unspecified
+// Returns: Status::OK() on success, -1 otherwise
+Status ByteSwapBuffer(void* buff, size_t size, DataType dtype, int num_of_elem);
```

Comment:
I don't believe ByteSwapBuffer will be useful outside of byte_swap.cc, can you omit this declaration in the header file and place the function definition in an unnamed namespace in byte_swap.cc? https://google.github.io/styleguide/cppguide.html#Internal_Linkage

Reply:
OK, I've placed the function definition in an unnamed namespace in byte_swap.cc.